stories/documentation/navigation/table-of-content/table-of-content-basic.stories.ts

```diff
@@ -1,45 +1,37 @@
 import { Meta, Story } from '@storybook/angular';
 
 interface TableOfContentBasicStory {
-	mod: string;
 }
 
 export default {
 	title: 'Documentation/Navigation/TableOfContent/Basic',
-	argTypes: {
-		mod: {
-			options: ['', 'mod-grey'],
-			control: {
-				type: 'radio',
-			}
-		},
-	},
 } as Meta;
 
 function getTemplate(args: TableOfContentBasicStory): string {
-	const classes = [args.mod].filter(Boolean).join(' ');
 	return `
-	<nav class="tableOfContent ${classes}">
-		<a class="tableOfContent-item is-active">Section 1</a>
-		<a class="tableOfContent-item">Section 2</a>
-		<a class="tableOfContent-item">Section 3</a>
-		<a class="tableOfContent-item">Section 4</a>
+	<nav class="tableOfContent">
+		<ul class="tableOfContent-list">
+			<li>
```

Comment:
Wouldn't the `list-items` be on the links instead?
(The links would become `list-item-actions`?)